docs/content/deployment/guides/running-locally.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Running Dagster Locally | Dagster"
+description: How to run Dagster on your local machine.
+---
+
+# Running Dagster Locally
+
+The easiest way to start up Dagster services during local development is to run:
+
+```shell
+dagster dev
+```
+
+This command launches both [Dagit](/concepts/dagit) and the [Dagster daemon](/deployment/dagster-daemon) locally, allowing you to start a full deployment of Dagster from the command line. Once started, the process should be kept running.
+
+## Locating your code
+
+There are a few ways that you can tell Dagster how to find the Python code containing your assets and jobs, which are outlined in the following tabs. If you've used the [dagster command line to create a project](/getting-started/create-new-project#bootstrapping-a-new-project) to create a project, or are using a Dagster example or quickstart repository, you can simply run the `dagster dev` command in the same folder as the project in order to load that code.
+
+<TabGroup>
+<TabItem name="From a file">
+
+Dagster can load a file directly as a code location. In the following example, we used the `-f` argument to supply the name of the file:
+
+```shell
+dagster dev -f my_file.py
+```
+
+This command loads the definitions in `my_file.py` as a code location in the current Python environment
+
+You can also include multiple files at a time:
+
+```shell
+dagster dev -f my_file.py -f my_second_file.py
+```
+
+---
+
+</TabItem>
+<TabItem name="From a module">
+
+Dagster can also load Python modules as code locations. When this approach is used, Dagster loads the definitions defined at the top-level of the module, in a variable containing the <PyObject object="Definitions" /> object of its root `__init__.py` file. As this style of development eliminates an entire class of Python import errors, we strongly recommend it for Dagster projects deployed to production.
```

Comment:
So this is an interesting case of verbiage. This paragraph says modules but is describing the behavior of installed modules, which is a subset of modules. `-m` can import vanilla files actually (just omit the .py)

Reply:
ah I just copied this over from the "deploying the daemon locally" docs, but can tweak